src/ert/ensemble_evaluator/snapshot.py

```diff
@@ -270,7 +293,7 @@ def from_cloudevent(self, event: CloudEvent) -> "PartialSnapshot":
                 for (
                     forward_model_id,
                     forward_model,
-                ) in self._snapshot.get_forward_models_for_real(
+                ) in based_on.get_forward_models_for_real(
```

Comment:
Can a realization timeout if there is no `based_on` argument provided?

Reply:
No, I have added an empty snapshot as default